mne/gui/__init__.py

```diff
@@ -35,13 +35,17 @@ def coregistration(tabbed=False, split=True, scene_width=0o1, raw=None,
         unnecessary for wx backend).
     scene_width : int
         Specify a minimum width for the 3d scene (in pixels).
-    raw : None | str(path)
-        Path to a raw file containing the digitizer data.
+    inst : None | str(path)
+        Path to an instance file containing the digitizer data. Compatible for
+        Raw, Epochs, and Evoked files.
     subject : None | str
         Name of the mri subject.
     subjects_dir : None | path
         Override the SUBJECTS_DIR environment variable
         (sys.environ['SUBJECTS_DIR'])
+    raw : str(path)
+        This parameter has been deprecated for the `inst` parameter.
+        `raw` will be removed in 0.11.
```

Comment:
You should remove this, and instead add this function name to `_deprecation_ignores` here:

https://github.com/mne-tools/mne-python/blob/master/mne/tests/test_docstring_parameters.py#L59

We don't want to have the deprecated parameters in the docstring anymore, they're just in the signature itself for completeness.